inc/dbmysql.class.php

```diff
@@ -691,8 +691,8 @@ static function isMySQLStrictMode(&$msg) {
       global $DB;
 
       $msg = 'STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_ZERO_DATE,NO_ZERO_IN_DATE,ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER';
-      $req = $DB->request("SELECT @@sql_mode as mode");
```

Comment:
huh? `DB::request()` should handle this properly; according to the method documentation